docs/genai/howto/build-models-for-snapdragon.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Build models for Snapdragon
+description: How to build model assets for Snapdragon devices
+has_children: false
+parent: How to
+grand_parent: Generate API (Preview)
+nav_order: 4
+---
+
+# How to build model assets for Snapdragon NPU devices
+
+These instructions demonstrate generating the Llama 3.2 3B model. You can use the same instructions to generate the Phi-3.5 mini instruct model.
+
+## Setup and pre-requisites
+
+1. Sign up for [Qualcomm AI Hub](https://aihub.qualcomm.com/) access
+
+   Once signed up, configure your Qualcomm AI hub API token
+
+   Follow instructions shown here: https://app.aihub.qualcomm.com/docs/hub/getting_started.html#getting-started
+
+2. Install the [Qualcomm AI Engine Direct SDK](https://softwarecenter.qualcomm.com/#/catalog/item/qualcomm_neural_processing_sdk_public)
```

Comment:
we point to https://qpm.qualcomm.com/main/tools/details/qualcomm_ai_engine_direct in the ORT QNN EP page. should we align on this?
This link only contains limited version and is a zip instead of installation package.